### PR TITLE
4.0 Release Changes

### PR DESCRIPTION
**Pull request by thomaspleasance**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,90 +4,83 @@ x-labels: &k8s-labels
...
+      - CCDC_LICENSING_CONFIGURATION=${CCDC_LICENSING_CONFIGURATION}
+    image: ccdcrepository.azurecr.io/onsite/ccdc-python-gateway:4.0.0
+    restart: unless-stopped
+
+  ccdc-identity:
+    image: ccdcrepository.azurecr.io/onsite/ccdc-identitygateway:4.0.0
     environment:
-      - CsdRepository__ServiceLocation=webcsdbackend
-      - CspRepository__ServiceLocation=webcsd-theory
+      - "ConnectionStrings__DefaultConnection=${CSD_DB_CONNECTIONSTRING};SearchPath=identity;"
```
Comment (hudson-ccdc) [inline docker-compose.yml:33]: Is this right? identity will be in the same database as CSD Data?

File: sample.env [hunk 2]
```diff
@@ -4,8 +4,9 @@
 CCDC_LICENSING_CONFIGURATION=la-code;123456-123456-123456-123456-123456-123456;
 
 # Password for local database
-CSD_DB_PASSWORD=A password of your choosing
-CSD_CACHE_PASSWORD=A password of your choosing
+CSD_DB_CONNECTIONSTRING=Server=database-server;Port=5432;Database=csd-database;User Id=postgres;Password=passwordhere
```
Comment (hudson-ccdc) [inline sample.env:7]: There is another branch here that manages database config, did we want to use it? or forget it?

https://github.com/ccdc-opensource/on-site-webcsd/blob/generic-database-config/sample.env

File: docker-compose.yml [hunk 3]
```diff
@@ -4,90 +4,83 @@ x-labels: &k8s-labels
...
+      - "PlatformClientSettings__RedirectUri=${PUBLIC_URI}/authentication/login-callback"
+      - "PlatformClientSettings__PostLogoutRedirectUri=${PUBLIC_URI}/authentication/logout-ca"
```
Comment (hudson-ccdc) [inline docker-compose.yml:35]: Shouldn't this be "PlatformClientSettings__PostLogoutRedirectUri=${PUBLIC_URI}/authentication/logout-callback"?

File: docker-compose.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (thomaspleasance) [reply docker-compose.yml:34]: Should be set per documentation not part of the main docker-compose file

File: docker-compose.yml (hunk 3 above)
Comment (thomaspleasance) [reply docker-compose.yml:36]: Ta will fix

File: docker-compose.yml (hunk 1 above)
Comment (thomaspleasance) [reply docker-compose.yml]: Nope have updated :)

File: sample.env (hunk 2 above)
Comment (thomaspleasance) [reply sample.env:7]: @hudson-ccdc could we have a chat about this as I'm unsure what your trying to say

Comment (thomaspleasance) [reply sample.env:7]: I have deleted the branch

File: docker-compose.yml [hunk 4]
```diff
@@ -1,93 +1,91 @@
...
+      kompose.service.type: LoadBalancer
+      kompose.service.expose: "true"
+    depends_on:
+      - ccdc-identity
+      - webcsd
+      - csd-python-gateway
+
+  csd-python-gateway:
+    environment:
+      - PlatformSecurity__DisableSSLValidation=true
```
Comment (chriluos) [inline docker-compose.yml:24]: As discussed, adding a comment here so it's not missed: 
Should this be set to true by default? If a user wants to use SSL validation, we should set this to false as this isn't recommended for prod usage.

**Review by hudson-ccdc — APPROVED** [review]

**Review by eyeboah-duako-ccdc — APPROVED** [review]

**Pull request merged**